integration-tests/tests/mocked/get-question/step-functions-local.test.ts

```diff
@@ -0,0 +1,107 @@
+import { HistoryEvent } from "@aws-sdk/client-sfn";
+import { SfnContainerHelper } from "./sfn-container-helper";
+
+jest.setTimeout(30_000);
+
+describe("step-function-local", () => {
+  describe("HMRC KBV Get Questions", () => {
+    let stnContainerHelper: SfnContainerHelper;
+    describe("is successful", () => {
+      beforeAll(async () => {
+        stnContainerHelper = new SfnContainerHelper();
+      });
+      afterAll(async () => stnContainerHelper.shutDown());
+      it("has a container is running", async () => {
+        expect(stnContainerHelper.getContainer()).toBeDefined();
```

Comment:
We should probably create a Jira ticket to address this

Reply:
https://govukverify.atlassian.net/browse/OJ-1897